specs/gloas/validator.md

```diff
@@ -187,12 +187,12 @@ The validator creates `payload_attestation_message` as follows:
 
 - If the validator has not seen any beacon block for the assigned slot, do not
   submit a payload attestation. It will be ignored anyway.
-- Set `data.beacon_block_root` be the HTR of the beacon block seen for the
-  assigned slot
+- Set `data.beacon_block_root` be the hash tree root of the beacon block seen
+  for the assigned slot
```

Comment:
Can we add punctuation (a period at the end) for the sentence too? For consistency.